Dropdown/Dropdown.js

```diff
@@ -299,7 +319,7 @@ const DropdownBase = kind({
 			{},
 			{},
 			{},
-			{childComponent: Item, itemProps: {size: 'small'}},
+			{childComponent: Item, itemProps: {size: 'small', css}, selectedProp: 'selected'},
```

Comment:
I think that if you don't apply `css` to the child Item, you will not need to add all the .item styling in Dropdown.module.less and this will not affect the fix for the bug described in this PR

Reply:
I had to apply it, as Item has accent colored border for selected. And I had to overwrite some style.